anitya/templates/logs.html

```diff
@@ -94,11 +94,11 @@
 
 </div>
 
-
-<table>
+<br>
```

Comment:
Sounds like a better plan for me yes, but that should be added in the CSS directly not in the html